inference-engine/src/mkldnn_plugin/nodes/extract_image_patches.cpp

```diff
@@ -127,7 +127,7 @@ class ExtractImagePatchesImpl : public ExtLayerBase {
             outputs[0]->getTensorDesc().getBlockingDesc().getOffsetPadding();
 
         const auto& inDims = inputs[0]->getTensorDesc().getDims();
-        const size_t inDimsSize = inDims.size();
+        const size_t inDimsSize = inDims.size(); // Must always be 4 according to the specs.
```

Comment:
Do we need some kind of checks for in & out num. dimensions and shapes? Their values are strictly defined in [specs](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/ExtractImagePatches_3.md).